### ISSUE TITLE
Define Rule Watermark Path

### ISSUE THREAD

**Issue by massimone73** [ntmcminn/alfresco-pdf-toolkit#27]
Hello.
I can't create a folder rule with Pdf Watermark.
What is the syntax to specify the exact path of the image file?
Watermark image: ?????
Thanks.

**Comment by gody01** [comment]
This is still issue with 5.1 & 5.2 version of Alfresco.
On linux there is no select box for picking image file.

**Comment by douglascrp** [comment]
Hello all.

The action is not ready to be used as a folder rule action because I guess this addon has not been created to be used like that, because most (if not all) the actions delivered by it are dependent on the current document being used, and depending on user's input.

In my opinion, the right thing to do is not to show the actions at all as options inside the folder rule management form.

@OrderOfTheBee/addons What do you think?

**Comment by gody01** [comment]
PdfToolkit is definitly very useful with rule actions.
One can use it as part of simple workflow to achieve many neat goals.

**Comment by douglascrp** [comment]
@gody01 Indeed, but when I need something like that, usually I go with the execute javascript, and I put all my logic inside of it.

**Comment by gody01** [comment]
With rule, there is no programming. Very useful for business power user.
I just hope, that there would be no removing of existing features, because there is a bug in one of options.
